### PR TITLE
Dev i18n

### PR DESCRIPTION
**Pull request by benlapETS**
Completed the task which consisted of #12:

* Using pybabel and gettext for making the application billingual (french/english)
* Make it available for Linux and Windows
* Don't break anything
* Document the work required to do so

WARNING THIS PR SHOULD BE RESOLVED **AFTER** #11

### CODE REVIEW

File: hanggame/locales/fr_CA/LC_MESSAGES/hanggame.po [hunk 1]
```diff
@@ -0,0 +1,195 @@
...
+msgid "nope"
+msgstr "nan"
+
+#: game.py:70
+msgid "not at all"
+msgstr "pas du tout"
+
+#: game.py:70
+msgid "fuck off"
+msgstr "va te faire foutre"
```
Comment (sriendea) [inline hanggame/locales/fr_CA/LC_MESSAGES/hanggame.po:78]: Not professionnal

File: hanggame/locales/fr_CA/LC_MESSAGES/hanggame.po [hunk 2]
```diff
@@ -0,0 +1,195 @@
...
+msgid "Sorry, you were wrong :_("
+msgstr "Désolé, vous vous êtes trompé :_("
+
+#: greeter.py:73
+msgid "Good guess! Keep it up!!"
+msgstr "Bon choix! Continuez comme ça!!"
+
+#: greeter.py:73
+msgid "Wow! you're strong!!"
+msgstr "Hou la la! vous êtes fort!!"
```
Comment (sriendea) [inline hanggame/locales/fr_CA/LC_MESSAGES/hanggame.po:142]: Hou là là ...

File: hanggame/locales/fr_CA/LC_MESSAGES/hanggame.po [hunk 3]
```diff
@@ -0,0 +1,195 @@
...
+msgid "Try another letter plz"
+msgstr "Essayez une autre lettre svp"
+
+#: greeter.py:83
+msgid "Welcome to The Hangman Game by labesoft"
+msgstr "Bienvenue dans le Jeu du Pendu par labesoft"
+
+#: greeter.py:84
+msgid "Congrats!! You have guessed the word correctly.."
+msgstr "Félicitations!! Vous avez bien deviné le mot."
```
Comment (sriendea) [inline hanggame/locales/fr_CA/LC_MESSAGES/hanggame.po:194]: Félicitation!! without the (s)

**Review by sriendea — COMMENTED** [review]
Better than ever

File: hanggame/locales/fr_CA/LC_MESSAGES/hanggame.po (hunk 1 above)
Comment (benlapETS) [reply hanggame/locales/fr_CA/LC_MESSAGES/hanggame.po:78]: ... but funny, isn't it ?

File: hanggame/locales/fr_CA/LC_MESSAGES/hanggame.po (hunk 2 above)
Comment (benlapETS) [reply hanggame/locales/fr_CA/LC_MESSAGES/hanggame.po:142]: Fixed

**Pull request merged**